pom.xml

```
@@ -5,10 +5,50 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.kie.trustyai</groupId>
     <artifactId>trustyaiexplainability</artifactId>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>0.3.0</version>
     <name>TrustyAI Explainability</name>
+    <description>TrustyAI is a java library for Explainable AI.</description>
+    <url>https://repository.jboss.org/nexus/content/repositories/trustyai-explainability/</url>
     <packaging>pom</packaging>
 
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+            <comments>At least one license must be present. Here is an example of ASL v. 2.0</comments>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:git@github.com:trustyai-explainability/trustyai-explainability.git</connection>
+        <url>scm:git:git@github.com:trustyai-explainability/trustyai-explainability.git</url>
+        <developerConnection>scm:git:git@github.com:trustyai-explainability/trustyai-explainability.git</developerConnection>
+    </scm>
+
+    <developers>
+        <developer>
+            <id>ruivieira</id>
+            <name>Rui Vieira</name>
+            <organization>Red Hat</organization>
+        </developer>
+        <developer>
+            <id>tteofili</id>
+            <name>Tommaso Teofili</name>
+            <organization>Red Hat</organization>
+        </developer>
+        <developer>
+            <id>RobGeada</id>
+            <name>Rob Geada</name>
+            <organization>Red Hat</organization>
+        </developer>
+        <developer>
+            <id>danielezonca</id>
+            <name>Daniele Zonca</name>
+            <organization>Red Hat</organization>
+        </developer>
```

We can probably also just link GH team page like
```suggestion
        <developer>
            <name>TrustyAI Explainability team</name>
            <url>https://github.com/orgs/trustyai-explainability/people</url>
        </developer>
```

Note: we need to make each of us `Public` in that page to show up